pgbouncer/tests/test_pgbouncer_integration_e2e.py

```diff
@@ -67,8 +67,11 @@ def assert_metric_coverage(version, aggregator):
     aggregator.assert_metric('pgbouncer.pools.sv_tested')
     aggregator.assert_metric('pgbouncer.pools.sv_login')
     aggregator.assert_metric('pgbouncer.pools.maxwait')
+    aggregator.assert_metric('pgbouncer.pools.maxwait_us')
     aggregator.assert_metric('pgbouncer.stats.avg_recv')
     aggregator.assert_metric('pgbouncer.stats.avg_sent')
+    aggregator.assert_metric('pgbouncer.stats.total_wait_time')
+    aggregator.assert_metric('pgbouncer.stats.avg_wait_time')
```

Comment:
You'll want to move those three assertions in the following else block.
Those metrics are only available with pgbouncer 1.8+ and because we test with multiple versions the tests are not passing for older versions.

Reply:
Thanks I think I got that taken care of now